notifier/selfstate/selfstate.go

```diff
@@ -24,19 +24,20 @@ type SelfCheckWorker struct {
 	Notifier   notifier.Notifier
 	Config     Config
 	tomb       tomb.Tomb
-	Heartbeats []heartbeat.Heartbeater
+	heartbeats []heartbeat.Heartbeater
+}
+
+// NewSelfCheckWorker creates SelfCheckWorker.
+func NewSelfCheckWorker(logger moira.Logger, database moira.Database, notifier notifier.Notifier, config Config) *SelfCheckWorker {
+	heartbeats := createStandardHeartbeats(logger, database, config)
+	return &SelfCheckWorker{Logger: logger, Database: database, Notifier: notifier, Config: config, heartbeats: heartbeats}
 }
 
 // Start self check worker
 func (selfCheck *SelfCheckWorker) Start() error {
-	if !selfCheck.Config.Enabled {
-		selfCheck.Logger.Debugf("Moira Self State Monitoring disabled")
-		return nil
-	}
 	senders := selfCheck.Notifier.GetSenders()
 	if err := selfCheck.Config.checkConfig(senders); err != nil {
-		selfCheck.Logger.Errorf("Can't configure Moira Self State Monitoring: %s", err.Error())
```

Comment:
why should we drop this log?

Reply:
We have log [here](https://github.com/moira-alert/moira/pull/807/files/af69b6f16d563eddd8de07ef1841233985734c0e#diff-2885fc3fa6525de656c944e024f6b45e1423a26aa3a00d798de5f7ffa52a403cR102) and it is enough

Reply:
not bad, super